scripts/run-integration-tests.sh

```diff
@@ -66,15 +68,22 @@ patch_excluded_tests() {
     fi
 }
 
-build_and_install_driver() {
-    log_info "Building exasol driver $driver_jar..."
-    local plugin_dir="$metabase_dir/plugins"
-    cd "$exasol_driver_dir"
-    clojure -X:build :project-dir "\"$(pwd)\""
-    ls -lah "$driver_jar"
-    log_info "Installing exasol driver $driver_jar to $plugin_dir"
-    mkdir -p "$plugin_dir"
-    cp -v "$driver_jar" "$plugin_dir"
+install_jdbc_driver() {
+    if [ ! -d "$metabase_plugin_dir" ]; then
```

Comment:
I know, I am picky. But this will cause an error if the entry exists in the file system, but is not a directory. On the other hand `mkdir` will probably print a useful-enough error message in that case.

Reply:
Done.